data/free worlds start.txt

```diff
@@ -1615,6 +1821,116 @@ mission "FW Pirates 4.1"
+			`A few loud bangs on the door interrupt JJ. Freya enters, and the two have a whispered conversation as you slowly wipe a few flecks of spit off your face. Then JJ straightens and gestures to the military policemen, who pull you to your feet. You walk to the airlock. 
+				goto slaves
+			label argue
+			`	Freya purses her lips. "The charge against you? Unsanctioned military action. You did not have permission to attack the pirates.`
+			`"Oh really? So every other time I've fought pirates, I was committing a crime? Why did you wait until now to arrest me?"`
+			`"Don't try to cloud the issue," JJ snaps. "Your strike force attempted to annex Greenrock against the will of its people. We don't do that; it's why we're 'The Free Worlds.' You wanted to be a goddamn hero, and you betrayed what we're supposed to stand for. We have written confessions from the other surviving captains, so don't even try to play innocent with us. Don't try blaming Tomek, either. You knew his order was illegal. The whole strike force knew it!" He motions to the policemen, who turn you around and begin restraining you.`
+			`"Yes, I knew," you say. "That's why I disobeyed him.`
+			`Katya motions the policemen to stop. "What did you say?"`
+			`"I disobeyed him. Tomek was drunk, and annexing Greenrock was an idiotic idea. They're pirates and anarchists. Their way of life depends on piracy and slavery. So I took away their way of life."`
+			`"You... you... what?" JJ sputters, "the only way to do that would be to bombard the planet! What kind of maniac are-"`
+			`"I'm the kind of maniac who's very selective about my targets. I levelled their spaceport facilities, their shipyards, their outfitters, their fuel depots, and their hangers." You call up images of Greenrock before and after your attacks`
+			`Katya goes pale. "You destroyed the slave yards?! How many innocent people lived in those-"`
+			`"-Thousands." You interrupt, and you gesture to the ship's communications system. "And they're listening to this conversation. In fact," you check a small viewscreen, "It looks like quite a few have gathered around my <ship>"
+				goto slaves
+			label slaves
+			`	JJ opens the door and gasps. A crowd of freed slaves has gathered. JJ stares at you, then the crowd, then back at you.`
+			`	You shrug. 'As I was trying to tell you, I freed Greenrock's slaves. Pretty near all of them are in those ships I brought back. Plust most of their former masters' wealth, at least what we could carry away."`
+			`	JJ glares at you. "So you're dumping these refugees on us and taking all that wealth? I don't want to trade one pirate threat for a-"`
+			`"ENOUGH!" you roar. "The captured freighters aren't mine. They're theirs. So is everything inside them. Now I've answered your questions, and I'll answer any more that you have, but I'll be damned if I ever let you insult me on my own ship again!"`
```

Comment:
there is a problem here
personality is left to the player, never decisively stated by the game